docs/src/upgrade/release-3.2.x-incubating.asciidoc

```diff
@@ -152,6 +152,15 @@ there is no need to do that manually.
 
 See: link:https://issues.apache.org/jira/browse/TINKERPOP-790[TINKERPOP-790]
 
+Gremlin-server.sh and Init Scripts
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+`gremlin-server.sh` is now also an init script and can no longer be started without parameters. To start it in the foreground, please use the `console` parameter. Also, it will still start in the foreground when provided a yaml configuration file.
+
+How to install as a service has been added to the reference docs.
+
+See: link:https://issues.apache.org/jira/browse/TINKERPOP-980[TINKERPOP-980]
```

Comment:
I would add a link to the section in the reference docs here in the "See:" section like was done [here](https://github.com/apache/tinkerpop/blob/master/docs/src/upgrade/release-3.2.x-incubating.asciidoc#graphson-20)